test/integration/react-streaming-and-server-components/test/index.test.js

```diff
@@ -400,6 +400,19 @@ async function runBasicTests(context, env) {
     expect(imageTag.attr('src')).toContain('data:image')
   })
 
+  it('should handle various exports', async () => {
```

Comment:
```suggestion
  it('should handle multiple named exports', async () => {
```

Small nit to make it more clear.